app/javascript/dashboard/components/CustomAttribute.vue

```diff
@@ -188,6 +188,9 @@ export default {
     urlValue() {
       return isValidURL(this.value) ? this.value : '---';
     },
+    hrefURL() {
+      return isValidURL(this.value) ? this.value : '';
```

Comment:
Should we sanitise the input on the server where it is committed to the model instead of having a front-end only solution? 
CC: @sojan-official

Reply:
My thoughts align with this article: https://benhoyt.com/writings/dont-sanitize-do-escape/
We could have URL validations for the backend. But it will also be safe to ensure the output is escaped.

Reply:
Yep, we can have both. I suggested this because folks may access this data via an API to display in their apps, and would assume that the data here is safe, we can perhaps escape it while sending it in the json

Reply:
@sojan-official what do we do here? I can approve this PR for now, should we make the backend changes too?